src/modules/dashboard/gameDev/gameDevHandlers/achievementGameDevHandlers.ts

```diff
@@ -30,6 +32,26 @@ export const getAchievementsHandler: Lifecycle.Method = async (request, h) => {
   }
 };
 
+export const getAchievementProgressHandler: Lifecycle.Method = async (request, h) => {
+  if (!request.params.achievementId) {
+    throw Boom.notFound('Invalid achievementId');
+  }
+  const achievement = await findAchievementById(request.params.achievementId);
+
+  if (!achievement) {
+    throw Boom.notFound('achievement not found');
+  }
+  const MAX_RES_SIZE = 100;
+  const achievementUnlockedList = await getAchievementUnlockedFromAchievement(
+    achievement,
+    MAX_RES_SIZE
+  );
```

Comment:
not sure I understood why 100 was added here. the limit value should come from the queryString param
ex: when making a GET /leaderboards/{id}/results?limit=123

Reply:
the limit value that is sent in the request is used through `request.query.limit`